flows/interfaces.go

```diff
@@ -411,6 +413,7 @@ type FlowRun interface {
 	LogEvent(Step, Event)
 	LogError(Step, error)
 	Events() []Event
+	ReceivedInput() bool
```

Comment:
can use this in mailroom where we calculate this to set `responded` in the database